### ISSUE TITLE
Hardcoded keys

### ISSUE THREAD

**Issue by dcoder2099** [acmacalister/Github-Swift#1]
https://github.com/acmacalister/Github-Swift/blob/master/Github/Login/Login.swift#L49-L50

**Comment by acmacalister** [comment]
@dcoder2099, agreed hardcoded keys are no bueno. This is was more of a test app to play with Swift against a real API and I don't have any plans to ship it to the app store.
